Governance Poll - March 18, 2024.md (#2593)
makerdao/community: governance/polls/GOV12.1.2 Bootstrapping Edit Proposal - March 18, 2024.md

---

> title: GOV12.1.2 Bootstrapping Edit Proposal - March 18, 2024

Reviewer: ```suggestion
title: Stability, Support, and Governance Scope Bootstrapping Edits - March 18, 2024
```

> summary: Signal your support or opposition to a GOV12.1.2 bootstrapping edit in order to prepare for the Launch Season of the Endgame Features.

Reviewer: ```suggestion
summary: Signal your support or opposition to the Stability, Support, and Governance Scope bootstrapping edits in order to prepare for the Launch Season of the Endgame Features.
```

discussion_link: https://forum.makerdao.com/t/gov12-1-2-bootstrapping-edit-proposal/23879
parameters:
    input_format: single-choice
    victory_conditions:
        - { type : plurality }
    result_display: single-vote-breakdown
version: v2.0.0
options:
   0: Abstain
   1: Yes
   2: No
start_date: 2024-03-18T16:00:00
end_date: 2024-03-21T16:00:00
---

> # Poll: GOV12.1.2 Bootstrapping Edit Proposal - March 18, 2024

Reviewer: ```suggestion
# Poll: Stability, Support, and Governance Scope Bootstrapping Edits - March 18, 2024
```
